resources/src/iosMain/kotlin/dev/icerock/moko/resources/ColorResourceExt.kt

```diff
@@ -22,3 +24,15 @@ fun ColorResource.getColor(userInterfaceStyle: UIUserInterfaceStyle): Color {
         }
     }
 }
+
+/**
+ * Returns null if no color asset is found in the ios app.
+ * update and configure copyColorAssetsToIOSApp in your ios run script to automate the copy process.
+ *       val copyColorAssetsToIOSApp = tasks.register<Copy>("copyColorAssetsToIOSApp") {
+ *              from("$rootDir/resources/build/generated/moko/iosMain/res/Assets.xcassets")
+ *              into("$rootDir/iosApp/iosApp/Assets.xcassets/colors")
+ *       }
+ */
+fun ColorResource.getThemeColor(): UIColor {
+    return UIColor.colorNamed(this.name)!!
```

Comment:
need to use `UIColor.colorNamed(this.name, inBundle = this.bundle)` - with it no any additional gradle tasks require

Reply:
fixed in #439